### ISSUE TITLE
r.http throws ReqlNonExistenceError

### ISSUE THREAD

**Issue by SAVE-POlNT** [rethinkdb/rethinkdb#6968]
hello , i installed rethinkdb with docker in one of my servers Ubuntu 18.04
and i wanted simply to get data from another api with r.http in python with the driver version 2.4.0
and in nodejs 12 with the lastest version of rethinkdb npm pachkage
i get the same error , i tried different versions btw
this is the error im getting with all the headers or the page : 
```Unhandled rejection ReqlNonExistenceError: Error in HTTP GET of `https://api.github.com/repos/rethinkdb/rethinkdb/stargazers`: unknown error when parsing response headers:
HTTP/2 200 ```

and this is my simple nodejs code 
```js

r = require('rethinkdb');
var c = null

r.connect({host: 'myhost, port: 28015,password:"mypass"}, function(err, conn) {
//console.log(r.http('myhost',header:"a").run(conn));
r.http('https://api.github.com/repos/rethinkdb/rethinkdb/stargazers',
       { method: 'GET',
         header: { 'Content-Type': 'application/json' } })
.run(conn)

});
```
i tested that the connection is working before removing the promisis
and it is the same error in both js and python

**Comment by srh** [comment]
I'm being obvious, but there is some problem around HTTP/2 support in libcurl (or in how we use it).

**Comment by SAVE-POlNT** [comment]
thank you for your answer
if this is the case , how do you explain the fact that i downgraded to version 2.3.5 of rethinkDB and http calls works normally , the docker container of v2.3.5 is 3 years old .

**Comment by srh** [comment]
It's probably using an older version of libcurl which doesn't support HTTP/2.

**Comment by srh** [comment]
This is addressed by #7012.

Went the lazy route of making Curl use http/1.1.
